jena-arq/src/main/java/org/apache/jena/riot/system/AsyncParser.java

```diff
@@ -199,7 +244,7 @@ protected X moveToNext() {
                         return null;
                     }
                     return x;
-                } catch (InterruptedException e) {
+                } catch (@SuppressWarnings("unused") InterruptedException e) {
```

Comment:
This `SuppressWarnings` causes a warning (an "info").

"At least one of the problems in category 'unused' is not analysed due to a compiler option being ignored"

The rest of the code does not use "unused".

Please remove.

Reply:
done